src/gui/entry/EditEntryWidget.cpp

```diff
@@ -178,6 +178,9 @@ void EditEntryWidget::setupMain()
 
     m_mainUi->expirePresets->setMenu(createPresetsMenu());
     connect(m_mainUi->expirePresets->menu(), SIGNAL(triggered(QAction*)), this, SLOT(useExpiryPreset(QAction*)));
+
+    // HACK: Align username text with other line edits. Qt does not let you do this with an application stylesheet.
+    m_mainUi->usernameComboBox->lineEdit()->setStyleSheet("padding-left: 8px;");
```

Comment:
I don't think this is an appropriate fix. The problem seems to be that one is a LineEdit and the other a ComboBox. Inconsistencies between these widget types should be fixed in the QStyle directly.

Reply:
I spent a long time trying to figure out exactly where the problem was. I dived into the BaseStyle and added padding in there and looked at all the exceptions. The problem is that QComboBox implements it's own QLineEdit and ignores ALL styling of QLineEdit applied. The only way to fix it is to override the style sheet post-construction. This was observed by many others in the Qt universe.